[core]/es_extended/client/modules/points.lua

```diff
@@ -0,0 +1,60 @@
+local Markers = {}
+
+---@func fun(coords: vector3, radius: number, data?: any)
+function ESX.CreateMarker(data) 
+    local Index = #Points+1
+    Markers[Index] = data
+    return Markers[Index]
+end
+
+CreateThread(function()
+    while true do 
+        Wait(0)
+        local InPoint = false
+        local Coords = GetEntityCoords(ESX.PlayerData.Ped)
+        for i = 1, #Markers do
+            local Marker = Markers[i]
+            Marker.distance = #(Coords - Marker.Coords) 
```

Comment:
Marker.Coords doesn't exist, and you should handle vector3. I mean convert if not vector 3.

Reply:
Done